bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -16,10 +16,46 @@
 else:
     pd = LazyLoader("pd", globals(), "pandas")
 
+try:
+    from typing import Literal
+except ImportError:  # pragma: no cover
+    from typing_extensions import Literal
+
 logger = logging.getLogger(__name__)
 
 
-class PandasDataFrame(IODescriptor):
+def _infer_type(item: str) -> str:  # pragma: no cover
+    if item.startswith("int"):
+        return "integer"
+    elif item.startswith("float") or item.startswith("double"):
+        return "number"
+    elif item.startswith("str") or item.startswith("date"):
+        return "string"
+    elif item.startswith("bool"):
+        return "boolean"
+    else:
+        return "object"
+
+
+def _schema_type(
+    mime_type: str, dtype: t.Optional[t.Union[bool, t.Dict[str, t.Any]]]
+) -> t.Dict[str, t.Any]:  # pragma: no cover
```

Comment:
why no cover?

Reply:
technically this should be covered in the tests, but it didn't for some reason